### PR TITLE
Statistics addition

### PR DESCRIPTION
**Pull request by BanalityOfSeeking**
Tried to add very basic statistics, (min, max, average, total).

### CODE REVIEW

**Comment by BanalityOfSeeking** [comment]
Add support for reporting frame diagnostics #30 Associated issue

**Comment by BanalityOfSeeking** [comment]
can't modify, resubmitting.

File: include/NovelRT/DebugService.h
Comment (RubyNova) [inline include/NovelRT/DebugService.h:33]: How come this is called getStatistics when it is a void returning method?

Comment (BanalityOfSeeking) [reply include/NovelRT/DebugService.h:33]: I chose get because I thought it fit into the update category, but I will change it maybe "setAccumalatedStatitistics"?

Comment (RubyNova) [reply include/NovelRT/DebugService.h:33]: If it is setting them then it makes sense - generally the name should reflect what it does after all. :D

**Review by RubyNova — APPROVED** [review]
I can't see anything wrong code-wise, I'll let some of the others chime in before merging though.

**Review by capnkenny — APPROVED** [review]
LGTM!

**Pull request merged**
